src/cephadm/cephadm

```diff
@@ -51,9 +51,9 @@ FuncT = TypeVar('FuncT', bound=Callable)
 DEFAULT_IMAGE = 'quay.io/ceph/ceph:v16'
 DEFAULT_IMAGE_IS_MASTER = False
 DEFAULT_IMAGE_RELEASE = 'pacific'
-DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
-DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v0.18.1'
-DEFAULT_ALERT_MANAGER_IMAGE = 'quay.io/prometheus/alertmanager:v0.20.0'
+DEFAULT_PROMETHEUS_IMAGE = 'registry.suse.com/ses/7/prometheus/prometheus-server:2.27.1'
+DEFAULT_NODE_EXPORTER_IMAGE = 'registry.suse.com/ses/7/prometheus/prometheus-node-exporter:1.1.2'
+DEFAULT_ALERT_MANAGER_IMAGE = 'registry.suse.com/ses/7/prometheus/prometheus-alertmanager:0.21.0'
```

Comment:
```suggestion
DEFAULT_ALERT_MANAGER_IMAGE = 'registry.suse.com/ses/7.1/prometheus/prometheus-alertmanager:0.21.0'
```

wondering if these should be under a `7.1` path or similar ?

Reply:
I took the `.../ses/7/...` path because that's where all other images lie. Or did I get anything wrong?

Reply:
I think when this PR was originally opened, we hadn't decided on 7 vs 7.1, but we've since decided to use the 7.1 paths.

Reply:
Yes, you're right! I'll need to update it.